### ISSUE TITLE
Audio cues no longer play

### ISSUE THREAD

**Issue by leonardder** [microsoft/vscode#157403]
Type: <b>Bug</b>

In the most recent insiders version, audio cues no longer play. I ensured that the audio cues I expect are enabled (e.g. error) and entered the following json as an example:

```json
{ hello }
```

I ensured that the error was reported when moving through errors with f8, yet the cue doesn't play when I come across the line with up/down arrow keys

VS Code version: Code - Insiders 1.71.0-insider (73fd3f11032e7b83c2ae011b5516e6ddd19e3db2, 2022-08-05T07:36:58.753Z)
OS version: Windows_NT x64 10.0.25151
Modes:
Sandboxed: No

**Comment by leonardder** [comment]
cc @isidorn

**Comment by hediet** [comment]
We didn't touch anything there the last months, maybe it is caused by an electron update. Will investigate for the next release.

**Comment by isidorn** [comment]
I can reproduce this. Easy repros steps:
1. F1 > list audio cues command
2. Audio cues do not produce sound 🐛  

Works fine on latest stable. Does not work on Insiders.
fyi @deepak1556 since most likely caused by Electron update. Maybe our future planned ffmpeg change will fix this?

@leonardder thanks a lot for reporting this, and sorry about this breakage.

**Comment by leonardder** [comment]
> @leonardder thanks a lot for reporting this, and sorry about this breakage.

You're welcome, I'm running insiders for a reason 😉
Looking forward to a fix.

**Comment by isidorn** [comment]
Let me also assign @hediet 
@hediet since we will now support `mp3` would you like that all our audio cues are in `mp3` format? If yes, would you like me to reach out to our sound designer for this? Thanks

**Comment by deepak1556** [comment]
@isidorn can you track the `mp3` conversion in a separate issue, this particular issue is a regression with the ffmpeg build shipped with internal builds. There was a recent build architecture change and it impacted this. I would like to close this issue as part of the next Electron version update :)

**Comment by isidorn** [comment]
@deepak1556 makes sense, thanks! There is actually already this one which we can use for the `mp3` conversion https://github.com/microsoft/vscode/issues/147190

**Comment by jvesouza** [comment]
The same problem using code-insiders into a fedora workstation 36.

**Comment by jvesouza** [comment]
It is working again with orca.

**Comment by isidorn** [comment]
@jvesouza lovely, thanks for confirming. Adding verified label.

**Comment by leonardder** [comment]
For me it also works like a charm on Windows.

**Comment by jvesouza** [comment]
For some reason I can't explain, this issue seems to have returned on my fedora workstation.
No sound is played when I move the cursor to lines that contain breakpoints, errors and etc.

Version: 1.71.0-insider
Commit: 28f604dec81b609dd7e69362585f7a99d4c48f88

**Comment by leonardder** [comment]
Confirmed on Windows. Could tests for this be added somehow?

**Comment by deepak1556** [comment]
Sorry for the regression, but this is a temporary expected breakage because we are updating our supported audio formats https://github.com/microsoft/vscode/pull/158486, audio cues will be updated to new format this iteration as part of https://github.com/microsoft/vscode/issues/147190

/cc @isidorn @hediet

**Comment by isidorn** [comment]
Sorry about that @jvesouza @leonardder. We just pushed a fix, and the VS Code insiders from tomorrow will have the fix.

**Comment by leonardder** [comment]
Thanks @isidorn 
Given this feature turns out to be pretty important for us visually impaired users, would it be possible to create a system test for this? I assume it should be possible to throw a audio cue at FFMPEG and make it fail if the sound doesn't play?

**Comment by isidorn** [comment]
@leonardder that's a good idea. However I leave this up to @hediet since he is driving this from engineering and would have better ideas how we could test it exactly.

**Comment by hediet** [comment]
We would need a way to ask the OS if an application played some sound.

**Comment by jvesouza** [comment]
It works again, at least on my environment.
Thanks.

**Comment by isidorn** [comment]
@jvesouza awesome, thanks for letting us know.

**Comment by leonardder** [comment]
I can confirm that it works here as well. Hopefully it stays that way 😉
